src/main/kotlin/com/software/modsen/driverservice/dto/response/DriverResponse.kt

```diff
@@ -3,7 +3,7 @@ package com.software.modsen.driverservice.dto.response
 import com.software.modsen.driverservice.model.Sex
 
 data class DriverResponse(
-    val driverId: Long?,
+    val id: Long?,
```

Comment:
Are you sure, that your id can be null in response?

Reply:
Fix:  Id cannot be null.

After getting an object from the database, it always has an id, so I decided to use an <id!!> in mapper